Src/Sarif.PartnerMatcher.Function/HttpSpamCheckFn.cs

```diff
@@ -0,0 +1,44 @@
+using System;
```

Comment:
you could merge this with the file `httpAnalyzeFn` and change the Run method Analyze or something like that.

Reply:
merged